src/sentry/release_health/metrics.py

```diff
@@ -599,7 +618,7 @@ def check_has_health_data(
         now: Optional[datetime] = None,
     ) -> Set[ProjectOrRelease]:
         if now is None:
-            now = datetime.now(pytz.utc)
+            now = datetime.now(timezone.utc)
```

Comment:
Since we are using `pytz.utc` in lots of other places. Can you please explain why did you decide to change it 🤔

Reply:
Mostly because it is just another unnecessary python library.
We have it for legacy reasons, but since 3.2 when `datetime.timezone` was introduced we don't need it anymore ( so I make a point to remove it in code that I touch).

Reply:
Yeah, it's a perfect point,
```
 rg "timezone.utc" | wc -l
     320
(.venv) ➜  sentry git:(metrics/metrics-layer-release-health-backend) ✗ rg "pytz.utc" | wc -l
     222
```
I'll make a note for myself to do the same; why don't we replace everywhere and remove dependency?